records_mover/db/bigquery/loader.py

```diff
@@ -32,9 +32,13 @@ def __init__(self,
         self.url_resolver = url_resolver
         self.gcs_temp_base_loc = gcs_temp_base_loc
 
+    def best_scheme_to_load_from(self) -> str:
+        return 'gs'
+
     @contextmanager
     def temporary_gcs_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
         if self.gcs_temp_base_loc is None:
+            # TODO: Can this yield None succesfully?  Would make logic more typesafe.
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker